packages/py-api/py_api/routes/questionnaires_routes.py

```diff
@@ -8,5 +8,5 @@
 
 
 @router.get("/csv/{dep}")
-async def get_csv(dep) -> StreamingResponse:
+async def get_csv(dep) -> StreamingResponse:  # type: ignore
```

Comment:
why are we ignoring the type here?

Reply:
I think this has been used to bypass commit errors.

Reply:
But adding dep: str fixed that issue making the #type ignore redundant